.github/workflows/test.yml

```diff
@@ -69,16 +71,16 @@ jobs:
 
       - run: npm install
 
-      - name: Setup
```

Comment:
maybe it makes sense to cache the downloaded mongo server to avoid downloading the mongo server again and again?

https://github.com/actions/cache

Reply:
> maybe it makes sense to cache the downloaded mongo server to avoid downloading the mongo server again and again?

true, will do once i figure out what the current CI problem is

Reply:
done in 518ee0e46ee655880b031f2e8fa5e3fc5427a27c, though it will not take effect until at least one CI passes